### ISSUE TITLE
Help needed in Arctos - bulk edit containers

### ISSUE THREAD

**Issue by bryansmclean** [ArctosDB/arctos#8065]
# Tell us what you are trying to do 

Bulkload containers with [reserved barcodes](https://github.com/ArctosDB/arctos/issues/8064)

# What are relevant pages in Arctos
**Provide a link to or a description of the page where you need help.**

[Create containers](https://handbook.arctosdb.org/how_to/How-to-Create-and-Edit-Containers.html)

When I attempt to load this .csv file

[container-cryobox-template.csv](https://github.com/user-attachments/files/16815586/container-cryobox-template.csv)

I get an "insufficient access" error (no other details)

**Comment by dustymc** [comment]
@bryansmclean you now have admin_container. 

Documentation is https://handbook.arctosdb.org/documentation/container.html 

That CSV looks like edit container, you need create: https://arctos.database.museum/loaders/bulkCreateContainer.cfm

Your collection does not yet have anything that could be edited:

```

arctosprod@arctos>> select container_type,count(*) from container where institution_acronym ='UNCG' group by container_type;
  container_type   | count 
-------------------+-------
 collection object |  3822
```

Let me know if you need anything!

**Comment by campmlc** [comment]
@bryansmclean I'm also happy to help. This process needs to be written down in the Workflows repository.

**Comment by bryansmclean** [comment]
> @bryansmclean I'm also happy to help. This process needs to be written down in the Workflows repository.

I agree. As a first-timer on containers, i'll just note that it was hard to figure out this top-level step (container create) since the links in the handbook arent all still active for code tables and Load CSV pages

**Comment by dustymc** [comment]
@campmlc PLEASE!

(See also https://github.com/ArctosDB/arctos/issues/8064#issuecomment-2321895671 for a point of confusion, and https://github.com/ArctosDB/arctos/issues/7466 for why it works that way.)

**Comment by bryansmclean** [comment]
Working on my end now, though. Thanks @dustymc.

**Comment by dustymc** [comment]
Thanks Bryan, transferring to docs for @campmlc
